### ISSUE TITLE
Cart Notification : Missing status message

### ISSUE THREAD

**Issue by mbambara** [Shopify/dawn#685]
**Current behavior:**
Status messages are not consistently included throughout the body of the theme and therefore do not meet WCAG 2.1 (Level AA) requirements. This specifically occurs when adding a product to the cart as there is no status update for this action.

**Expected behaviour:**
Status should be read on screen readers when an item is added to the cart.

**Version information:**
- Dawn Version: 2.1.0
- Chrome Version 93.0.4577.8
- macOS Version 10.14.6
